.github/workflows/CondaLock.yml

```diff
@@ -35,13 +35,13 @@ jobs:
           environment-file: environment-condalock.yml
 
       - name: Run conda-lock ${{ matrix.IMAGE }}
-        timeout-minutes: 20
+        timeout-minutes: 45
```

Comment:
@scottyhq, I've increased the timeout-minutes for `conda-lock` here to 45min, hopefully enough? (need to merge this into the main branch first before it'll actually work).